anthonyperry3/team-link-app
# Ab issue#25  added messaging functionality

## Changes
_List Changes Introduced by this PR_
1. Added the ability for 2 matched users to have a messaging conversation 


## Purpose
created a sendMessage function to create a 'messages' collection inside of matches by matchid and a useEffect with a listener (onSnapshot) queried so that all messages will be retrievable in desc order in realtime.


## Testing Steps
Test on 2 separate accounts that have matched and send messages to each other to confirm messages are correctly sent and displayed


_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #25
